python/sparkdl/graph/builder.py

```diff
@@ -83,15 +84,15 @@ def asGraphFunction(self, inputs, outputs, strip_and_freeze=True):
 
         :param inputs: list, graph elements representing the inputs
         :param outputs: list, graph elements representing the outputs
-        :param strip_and_freeze: bool, should we remove unused part of the graph and freee its values
+        :param strip_and_freeze: bool, should we remove unused part of the graph and free its values
```

Comment:
lol I think it's actually supposed to be `freeze` instead of `free` :P

Reply:
no idea how this happened :P